bin/sentieon-genomics-202010.02/lib/python/sentieon/vcflib/vcf.py

```
@@ -0,0 +1,465 @@
+        vals[5] = float(vals[5]) if vals[5] != '.' else None
+        vals[6] = vals[6].split(';') if vals[6] != '.' else []
+        end = info.get('END', vals[1] + len(vals[3]))
+        vals = vals[:7] + [info, samples, end, line]
+        return Variant(*vals)
+
+    @staticmethod
+    def genotypes(a, p):
+        if len(a) == 1:
+            return (a * p, )
+        a, b = a[:-1], a[-1:]
+        return [g + b*k for k in xrange(p+1) for g in VCF.genotypes(a, p-k)]
```

Ruff (0.8.2)

`xrange` may be undefined, or defined from star imports

(F405)